Sources/JNI/SwiftJNI.swift

```diff
@@ -215,7 +239,7 @@ extension JNI {
 
         var result = [JavaByte](repeating: 0, count: count)
         _env.pointee.pointee.GetByteArrayRegion(_env, array, jsize(startIndex), jsize(count), &result)
-        return result.map { UInt8(bitPattern: $0) } // may not have the same numeric value, but
+        return result.map { UInt8(bitPattern: $0) } // numeric value may change here because of the bitPattern constructor
```

Comment:
I need to revisit this, but i think there were some problems because UInt8 only allows positive values. Our midi data can be interpreted as negative values though, which caused some compiler errors.